project.yaml

```diff
@@ -18,6 +18,8 @@ actions:
       moderately_sensitive:
         monthly_deaths_place: output/os_reports/eol_service/deaths_month.csv
         monthly_deaths_place_plot: output/os_reports/eol_service/deaths_month_plot.png
+        eol_med_place: output/os_reports/eol_service/eol_med_pod.csv
```

Comment:
The name can be changed to 'monthly_meds_place' and 'monthly_meds_place_plot' in the yaml file

Reply:
Agreed. I think consistency is good and will help us.